opentelemetry-instrumentation/tests/auto_instrumentation/test_load.py

```
@@ -297,27 +299,36 @@ def test_load_instrumentors_dep_conflict(self, iter_mock, dep_mock):  # pylint:
         ep_mock4 = Mock()
         ep_mock4.name = "instr4"
 
+        dependency_conflict = DependencyConflict("1.2.3", None)
+
         distro_mock = Mock()
+        distro_mock.load_instrumentor.side_effect = [
+            None,
+            DependencyConflictError(dependency_conflict),
+        ]
 
         iter_mock.return_value = (ep_mock1, ep_mock2, ep_mock3, ep_mock4)
-        # If a dependency conflict is raised, that instrumentation should not be loaded, but others still should.
```

This is not valid anymore?

---

@emdneto, let me update the comment here. Its still valid, just throws an exception now instead of returning an object

---

@emdneto, comment added and expanded upon. Good catch. Thank you